kedro/framework/cli/pipeline.py

```diff
@@ -239,6 +239,11 @@ def pull_package(  # pylint:disable=unused-argument, too-many-arguments
     **kwargs,
 ) -> None:
     """Pull and unpack a modular pipeline in your project."""
+    deprecation_message = (
+        "DeprecationWarning: Command `kedro pipeline pull` is deprecated in Kedro 0.18.0. "
+        "Please use `kedro micropkg pull` instead."
+    )
+    click.secho(deprecation_message, fg="red")
```

Comment:
Let's not forget to remove these commands on `develop`! 🙌

Reply:
Will create a second PR to refactor these on `develop` once this lands.

Reply:
Yeah that's what I meant, definitely not here.